lib/api_transaction.go

```diff
@@ -137,3 +138,73 @@ func GetTransactionByHashHandler(storage *sebakstorage.LevelDBBackend) http.Hand
 		}
 	}
 }
+
+const GetMissingTransactionByHashHandlerPattern = "/missingtxs"
+
+func GetMissingTransactionsByHashHandler(storage *sebakstorage.LevelDBBackend /**nr *NodeRunner**/) http.HandlerFunc {
```

Comment:
Is the API need the streaming feature?
If pagination is possible then, IMHO, the streaming is not needed
or, client-side can handle it